7.2/rootfs/etc/php7/conf.d/50-setting.ini

```diff
@@ -7,3 +7,4 @@ memory_limit = 256M
 
 realpath_cache_size=4096K
 realpath_cache_ttl=600
+decorate_workers_output = no
```

Comment:
https://www.php.net/manual/en/install.fpm.configuration.php

```
decorate_workers_output boolean
Enable the output decoration for workers output when catch_workers_output is enabled. Default value: yes. Available as of PHP 7.3.0.
```

@jorge07 This setting should be removed from 7.2 version

Reply:
Totally right. Will submit a patch before release it. Thanks!